validators/crypto_addresses/eth_address.py

```diff
@@ -0,0 +1,55 @@
+"""ETH Address."""
+# -*- coding: utf-8 -*-
+
+# standard
+import re
+
+from sha3 import keccak_256
```

Comment:
Can't you use [`hashlib.sha3_256()`](https://docs.python.org/3/library/hashlib.html#hashlib.sha3_256)?

Reply:
No. The checksum validation is written based on `keccak_256` hash output.

Reply:
Instead of [`pysha3`](https://pypi.org/project/pysha3/), please use the official package: [`eth-hash`](https://github.com/ethereum/eth-hash).